### PR TITLE
Martial Art - Proxy Palm

### PR DESCRIPTION
**Pull request by Comxy**
## About The Pull Request
Hello, I am back with a new high quality PR.

This PR will add a new martial art obtainable through the palm clock available on the blackmarket.

https://user-images.githubusercontent.com/25363960/175333704-91a9970f-38c6-4cb9-bd70-b65b38f1a4ca.mp4
- 30% chance maximum of 1 palm cloak will be available.
- Shove Grab Punch. Stun your opponent.
- Punch Shove Punch Shove. Knocks down all opponents near you.
- Shove Grab Shove Grab. Makes you and your opponents switch places.
- Grabbing someone changes places with them. You will also have the ability to create wind blades.
- Fixes a reset bug for combo martial arts.

## Why It's Good For The Game
It is a cool new addition to the blackmarket. Now people might have another reason to get a blackmarket uplink and get cool stuff.

## Changelog

:cl:
add: New martial arts (proxy palm) and item to the blackmarket.
fix: Fixes combo reset bug.
soundadd: New proxy palm sound effects.
imageadd: New proxy palm cloak icons.
/:cl:

### CODE REVIEW

**Comment by calliophis-maculiceps** [comment]
is this a reference to anything/inspired by anything? sounds really cool

**Comment by Comxy** [comment]
> > Shove Grab Shove Grab. Makes you and your opponents switch places.
> > Grabbing someone changes places with them.
> 
> What's the difference?

The combo switches you and the people around you with someone else while doing stamina damage.

**Comment by Comxy** [comment]
> is this a reference to anything/inspired by anything? sounds really cool

This martial art is inspired by fighting multiple people. I though about what would work and wouldn't. So I kept it simple while still being potentially useful depending on how it is used.

**Comment by castawaynont** [comment]
martial arts 💀💀💀

File: code/datums/actions/mobs/projectileattack.dm [hunk 1]
```diff
@@ -325,3 +325,19 @@
...
+	background_icon_state = "bg_default"
+	desc = "Fires a wind blade at a target."
+	cooldown_time = 30 SECONDS
+	projectile_type = /obj/projectile/windblade
+	projectile_sound = 'sound/weapons/windclap.ogg'
+
+/datum/action/cooldown/mob_cooldown/projectile_attack/wind_clap/Activate(atom/target_atom)
+	. = ..()
+	playsound(owner, projectile_sound, 50, TRUE, -1)
+	owner.visible_message(span_danger("[owner] fires the proto-kinetic accelerator!"))
```
Comment (ATH1909) [inline code/datums/actions/mobs/projectileattack.dm:342]: 🤨

Comment (Comxy) [reply code/datums/actions/mobs/projectileattack.dm:342]: Oh shit whoops, almost everything is copied in some way.

**Comment by tralezab** [comment]
I don't like the fact that it's a boring martial art, in an uncompetitive item slot, acquired with a lame system

**Comment by zxaber** [comment]
>in an uncompetitive item slot

To it's credit, it also makes it very obvious when someone has the cloak on.

**Comment by Comxy** [comment]
> I don't like the fact that it's a boring martial art, in an uncompetitive item slot, acquired with a lame system

How is the martial art boring? It is unique and requires skill to use unlike carp which is mostly used for dodging bullets, or cqc where you can stunlock people into walls until they are dead. You can even charge up attacks on yourself at the cost of stamina.

I could change the slot if that is a problem. I made it a cloak because cloaks are cool.

I put it on the black market because it I thought it would add depth to the black market being able to get some unique items from it. I could add it to drone exploration instead.

**Comment by SuperNovaa41** [comment]
can you atomize the fix please

**Comment by Comxy** [comment]
Why did you close it?

**Comment by Shadowflame909** [comment]
mfw arm hates martial arts???

**Comment by PCPort** [comment]
Very disappointing to see this closed, tbh. More martial arts variety would be awesome to see

**Pull request closed without merging**